packages/cli/src/commands/inspect/index.ts

```diff
@@ -204,7 +204,8 @@ async function printDetails({
       customEnvironmentName && deployment.team?.slug
         ? `${link(
             `${target}`,
-            `https://vercel.com/${deployment.team.slug}/${name}/settings/environments`,
+            // @ts-ignore - customEnvironment needs to be defined on Deployment
```

Comment:
I would suggest `@ts-expect-error` instead. But also, can we add the type to `Deployment` so it's not necessary?